<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MO2 Separator Generator (Qt)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="editPanel" native="true">
      <layout class="QHBoxLayout" name="editPanelLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="editNameLabel">
         <property name="text">
          <string>Category Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editNameEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="editSaveBtn">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editCancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="colorBar">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="inputBar">
        <item>
         <widget class="QComboBox" name="typeCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="ofLabel">
          <property name="text">
           <string>of</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="catCombo"/>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="gradientLabel">
        <property name="text">
         <string>Category Separator Gradient:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="startColorLabel"/>
      </item>
      <item>
       <widget class="QLineEdit" name="startColorEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="startColorBtn">
        <property name="text">
         <string>Choose Start Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="endColorLabel"/>
      </item>
      <item>
       <widget class="QLineEdit" name="endColorEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="endColorBtn">
        <property name="text">
         <string>Choose End Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QPushButton" name="removeBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Remove Separator</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="addBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add Separator</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="genBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Generate Files</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QPushButton" name="moveUpBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Move ↑</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="QPushButton" name="moveDownBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Move ↓</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
